dashboard/client/src/components/ActorTable.tsx

```diff
@@ -39,14 +40,22 @@ export type ActorTableProps = {
   detailPathPrefix?: string;
 };
 
+const SEQUENCE = {
+  FIRST: 1,
+  MIDDLE: 2,
+  LAST: 3,
+};
+
 type StateOrder = {
   [key in ActorEnum]: number;
 };
+
 const stateOrder: StateOrder = {
-  [ActorEnum.ALIVE]: 0,
-  [ActorEnum.PENDING]: 1,
-  [ActorEnum.RECONSTRUCTING]: 2,
-  [ActorEnum.DEAD]: 3,
+  [ActorEnum.ALIVE]: SEQUENCE.FIRST,
+  [ActorEnum.DEPENDENCIES_UNREADY]: SEQUENCE.MIDDLE,
+  [ActorEnum.PENDING_CREATION]: SEQUENCE.MIDDLE,
+  [ActorEnum.RESTARTING]: SEQUENCE.MIDDLE,
```

Comment:
Why don't we have a specific order for all 5 values? Why just 3 orders?

Reply:
That's a good question. The reason is that we only need to consider the relative order between ALIVE and DEAD actors. In the pull request at https://github.com/ray-project/ray/pull/33395, we need to sort the actors so that the alive actors appear first and the dead actors appear last. For actors in any other states, we simply place them in the middle